tests/spec/core/data-cite-spec.js

```diff
@@ -372,4 +372,26 @@ describe("Core — data-cite attribute", () => {
     dfnA = doc.querySelector("#t2 > dfn > cite > a");
     expect(dfnA.href).toBe("https://fetch.spec.whatwg.org/");
   });
+
+  it("Adds <cite> around <a> when frag and path are missing from <a data-cite='some-spec'>", async () => {
+    const ops = {
```

Comment:
We also have a `makeStandardOps(body, respecConfigOverride)` function which gives more abstraction as:
``` js
const body = `
  <section>
    <p id="t1"><a data-cite="HTML"></a></p>
    <p id="t2"><a data-cite="Fetch"></a></p>
  </section>`;
const ops = makeStandardOps(body, null);
```

Reply:
Yeah I saw this structure in other test files but since most of the tests in data-cite were following the method that is there in this PR so I went with that. 😬 
Should I change it?

Reply:
Didn't know that! A refactoring for other tests would be good, especially when even active maintainers are frequently just copy-pasting existing tests.

Reply:
>  Should I change it?

Yes please.

Reply:
Should I change it in this PR itself or open a new one? I am okay with either of them.

Reply:
Lets keep the the fix for this PR here only, then we can do a refactor for other tests later in some other PR.

Reply:
Sure. Will do it. 🙂